pages/account/index.tsx

```diff
@@ -47,6 +47,58 @@ export default function MyAccount({
     }
   }
 
+  /** Testing settings api route */
+  async function settingsUpdate(patronId) {
+    try {
+      const response = await fetch(
+        `/research/research-catalog/api/account/settings/${patronId}`,
```

Comment:
Curious: Does this work if you remove the base `/research/research-catalog`?

Reply:
It does not .

Reply:
Update this to use the `BASE_URL` constant.